punwrap.cpp

```diff
@@ -27,6 +27,8 @@ double* qmap = NULL;
 double* phase;
 double* unwrapped;
 
+double _HUGE=1e37;
```

Comment:
I would add, if a variable is meant to never be modified you should write
```suggestion
const double _HUGE=1e37;
```
even better with C++11 use constexpr
```suggestion
constexpr double _HUGE=1e37;
```
I heavily recommend switching to C++11 for many of it's good new features. Static asserts, constexpr , nullptr, list initialisation, range-based for loops and ranges in general.
https://en.cppreference.com/w/cpp/11
It's not just fancy new words, it's a paradigm shift and a better safer language

Reply:
I know git quite well and am very comfortable with git.  However I am _not_ familiar with pull requests and github.  I mean I've used github quite a bit but I've only ever created one PR and it was just pulled with no fanfare.

I don't have a fork of DFTF.  I am working on it directly.  The changes for this pull request are in branch "gr".  I want to merge the changes back to master.  But I want Dale to agree.  What is the normal procedure for handling this PR.  Do I somehow tell this PR that it should be looking at branch gr?  Do I merge them back to master and *then* will github automatically realize that the PR has been merged?  I should go read about it but since you seem to know more, please let me know.  I could add the const.  Adding .gitignore is great.

Adding Makefile or moc_* files was never a problem for Dale as qt creator puts the generated files in a different folder.  But for linux users a .gitignore would be very helpful.  I'm happy to do those things but again, you are showing suggested changes on the wrong branch.  The "const" doesn't apply anymore, I believe.  That line of code is gone I believe.  90% sure.

How do I tell this PR to point to the latest gr branch instead of the brave-ulysses commit?

Reply:
@gr5  this would be a 5min talk on discord if you are interested (or 1h if we like the discussion)
I opened #8 for you with correct branch. The PR we are on, is brave_ulys forked and modified master branch.
![image](https://user-images.githubusercontent.com/4628382/217840286-af1517f2-92bf-4631-b28c-aa673ea118f8.png)

Maybe you can modify the PR, easyest is just to close and look at #8 only

Dale does not know PR and GitHub eather so I would recommend I help you on the standard "process" on open source. It might sound like I want it my way but I honestly think it's your best option. I worked on many repositories and even if you think I do lack the DFT fringe experience I will never harm your code without permission. However I like to interfer in the process thing.